arm-datalake-analytics/job/2015-11-01-preview/swagger/job.json

```diff
@@ -706,6 +706,10 @@
         },
         "type": {
           "type": "string",
+          "enum": [
+            "USql",
+            "Hive"
+          ],
```

Comment:
Did you have a look at the x-ms-enum extension doc over here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#Enum-x-ms-enum